test/cpp/end2end/crl_provider_test.cc

```diff
@@ -78,6 +78,14 @@ TEST(DirectoryReloaderCrlProviderTestNoFixture, Construction) {
   auto provider = grpc_core::experimental::CreateDirectoryReloaderCrlProvider(
       kCrlDirectoryPath, std::chrono::seconds(60), nullptr);
   ASSERT_TRUE(provider.ok()) << provider.status();
+  EXPECT_FALSE((*provider)->DenyUndetermined());
+}
+
+TEST(DirectoryReloaderCrlProviderTestNoFixture, DenyUndeterminedTrue) {
+  auto provider = grpc_core::experimental::CreateDirectoryReloaderCrlProvider(
+      kCrlDirectoryPath, std::chrono::seconds(60), nullptr, true);
```

Comment:
nit: Same here RE inline comment for literals.

Reply:
Done